test/parzival/companies_test.exs

```diff
@@ -12,7 +12,41 @@ defmodule Parzival.CompaniesTest do
 
     test "list_offers/0 returns all offers" do
       offer = offer_fixture()
-      assert Companies.list_offers() == [offer]
+
+      assert Companies.list_offers() ==
+               {:ok,
+                {[offer],
+                 %Flop.Meta{
+                   current_offset: 0,
+                   current_page: 1,
+                   end_cursor: nil,
+                   errors: [],
+                   flop: %Flop{
+                     after: nil,
+                     before: nil,
+                     filters: [],
+                     first: nil,
+                     last: nil,
+                     limit: 9,
+                     offset: nil,
+                     order_by: [:minimum_salary, :maximum_salary],
+                     order_directions: [:desc, :desc],
+                     page: nil,
+                     page_size: nil
+                   },
+                   has_next_page?: false,
+                   has_previous_page?: false,
+                   next_offset: nil,
+                   next_page: nil,
+                   page_size: 9,
+                   params: %{},
+                   previous_offset: nil,
+                   previous_page: nil,
+                   schema: Parzival.Companies.Offer,
+                   start_cursor: nil,
+                   total_count: 1,
+                   total_pages: 1
+                 }}}
```

Comment:
Well, this is some sorta of _cursed_ code 😅
Isn't there any other option?

Reply:
If you find one , please let me know , i also don't like that

Reply:
It's not the better approach, but `assert Companies.list_offers([]) == [offer]` should do the job.

Reply:
Done ! :+1: